rpcs3/rpcs3qt/CMakeLists.txt

```diff
@@ -154,4 +155,5 @@ target_link_libraries(rpcs3_ui
         3rdparty::libpng
         3rdparty::7z
         3rdparty::wolfssl
-        3rdparty::libcurl)
+        3rdparty::libcurl
+		3rdparty::rtmidi)
```

Comment:
```suggestion
        3rdparty::rtmidi)
```

Reply:
epic text editor moment

I'll fix this in the next commit as I figured out how to support the other type of guitar I wasn't emulating yet.